docs/apache-airflow/concepts.rst

```diff
@@ -1618,7 +1618,7 @@ do the same, but then it is more suitable to use a virtualenv and pip.
     mkdir zip_dag_contents
     cd zip_dag_contents
 
-    pip install --install-option="--install-lib=$PWD" my_useful_package
+    pip install --prefix=$PWD my_useful_package
```

Comment:
```suggestion
    pip install --target=$PWD my_useful_package
```

Re-reading the context, the zip file is loaded directly into `sys.path`, so I think it should use `pip install --target` instead. `--prefix` would give you a nested prefix structure (e.g. Python packages are installed into `$PREFIX/lib/pythonX.Y/site-packages`). `--target` gives you a flat structure that can be directly loaded into `sys.path`.